CompositionRoot/ModuleRegistrar.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Immutable;
+using Autofac;
+
+namespace Messerli.CompositionRoot
+{
+    public class ModuleRegistrar : Module
```

Comment:
This class can be `internal sealed`

Reply:
Addressed in: https://github.com/messerli-informatik-ag/composition-root/pull/4/commits/91d57120fbae0705c7c53cc9d129d8293abedc96

Reply:
Addressed in: https://github.com/messerli-informatik-ag/composition-root/pull/4/commits/f3adbe9ed8bd578935367eee0c033a8c3ebd359b